module-3/backend/week_one.md

```diff
@@ -5,7 +5,16 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is `json` and why is it important?
+  JSON stands for JavaScript Object Notation, it is important because APIs typically respond with JSON 
+  or XML. JSON being the easier of the two to work with.
 2. What's the difference between `joins` and `includes` in ActiveRecord?
+  Joins makes one table out of 2. Includes allows you to access specified attributes without making
+  additional queries to the database.
 3. What's an API?
+  API stands for Application Program Interface, it allows you to access another code bases database through
```

Comment:
we're accessing another application's endpoints - this might include getting access to the data - OR we could be sending data to the API. it depends!